rewrite-gradle/src/test/java/org/openrewrite/gradle/AddDependencyTest.java

```diff
@@ -1360,4 +1370,41 @@ private AddDependency addDependency(String gav, @Nullable String onlyIfUsing, @N
           (gavParts.length < 4) ? null : gavParts[3], null, null, null
         );
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"com.google.common.math.*", "com.google.common.math.IntMath"})
+    void onlyIfUsingTestScope(String onlyIfUsing) {
```

Comment:
Was this test just being logically moved to make it easier to see with respect to the new test?

Reply:
TBH idk